include/circt/Dialect/SV/SVExpressions.td

```diff
@@ -82,6 +82,28 @@ def VerbatimExprSEOp : SVOp<"verbatim.expr.se", [HasCustomSSAName]> {
   ];
 }
 
+def MacroRefExprOp : SVOp<"macro.ref", [HasCustomSSAName]> {
+  let summary = "Expression to refer to a SystemVerilog macro";
+  let description = [{
+    This operation produces a value by referencing a named macro.
+
+    Presently, it is assumed that the referenced macro is a constant with no
```

Comment:
Please add a SE (side-effecting) version of this op, similar to verbatim.

Reply:
This is the side-effecting version of the op. Does it require a specific trait?

Reply:
I will not do that in this PR since it is not needed yet.